### ISSUE TITLE
[FEATURE REQUEST]: User-defined configuration

### ISSUE THREAD

**Issue by rustatian** [spiral/roadrunner-cli#29]
Hey @butschster 😃 

Every time the user downloads a `RR` via the `./rr --get` command, it receives a complete configuration. It's hard to choose what is needed to run, for example, a simple HTTP worker. First of all, it's hard to determine who's responsible for the `HTTP` in the config because it has > 1.5k lines of YAML. By `HTTP` I mean not only that plugin. It's here to represent how users might be embarrassed when looking at that tremendous full configuration reference.

I propose to add the flags to the CLI application to build the configuration, including only specified by the user sections. Or, if no flags are provided, download the complete reference. For example:

1. `./rr get --http` will download only the following config:
```yaml
version: "2.7"
# Application configuration
rpc:
  listen: tcp://127.0.0.1:6001

server:
  command: "php worker.php"

http:
  address: "localhost:7233"
  max_request_size: 1000
  pool:
    num_workers: 5

logs:
  mode: development
  level: debug
  encoding: console
```

This is the minimum configuration needed to run a `worker.php` with HTTP support.
`server`, `logs`, `rpc` sections will be here by default for any combination of flags.

2. Next sample: `./rr get --http, temporal`
```yaml
version: "2.7"
# Application configuration
rpc:
  listen: tcp://127.0.0.1:6001

server:
  command: "php worker.php"

http:
  address: "localhost:7233"
  max_request_size: 1000
  pool:
    num_workers: 5

# Workflow and activity mesh service
temporal:
  address: "localhost:7233"
  activities:
    num_workers: 1

logs:
  mode: development
  level: debug
  encoding: console
```
Simple and understandable.

So basically, the flags are configuration sections.

**Comment by roxblnfk** [comment]
The flag like `--no-comments` (or `--without-comments`) can be also useful

**Comment by roxblnfk** [comment]
We can make also beautiful webform for RR configuring
